### PR TITLE
feat(mobile): display current refinements in filters panel

### PR DESCRIPTION
**Pull request by francoischalifour**
## Description

This PR changes the following on mobile:

- Move the current refinements in the filters panel
- Display the filters count in the filters button. This gives a clue to users about why only some search results appear.

## Preview

image

image

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -575,6 +575,14 @@
   width: 16px;
 }
 
+.uni-FiltersButton-Count {
+  background: rgba(255, 255, 255, 0.32);
+  border-radius: 10px;
```
Comment (sarahdayan) [inline src/App.scss:580]: Good trick to keep this fully round even if we change the size of the badge.

```suggestion
  border-radius: 9999px;
```

File: src/App.scss [hunk 2]
```diff
@@ -575,6 +575,14 @@
...
+  margin-left: 0.5rem;
+  min-width: 24px;
```
Comment (sarahdayan) [inline src/App.scss:582]: Is it necessary?

Comment (francoischalifour) [reply src/App.scss:582]: Yep because with short-width numbers (e.g., `1`) it looks weird when corders are too rounded.

File: src/App.scss [hunk 3]
```diff
@@ -575,6 +575,14 @@
   width: 16px;
 }
 
+.uni-FiltersButton-Count {
+  background: rgba(255, 255, 255, 0.32);
+  border-radius: 9999px;
+  margin-left: 0.5rem;
+  min-width: 24px;
```
Comment (francoischalifour) [inline src/App.scss:582]:
```suggestion
  min-width: 22px;
```

**Pull request merged**